diff --git a/node_modules/next-yak/loaders/cssloader.cjs b/node_modules/next-yak/loaders/cssloader.cjs
index 0c1417e..a0c7fe4 100644
--- a/node_modules/next-yak/loaders/cssloader.cjs
+++ b/node_modules/next-yak/loaders/cssloader.cjs
@@ -12,6 +12,7 @@ const {
   getConstantName,
   getConstantValues,
 } = require("./lib/getConstantValues.cjs");
+const babelPluginTypeScript = require('@babel/plugin-syntax-typescript');
 
 /**
  * @typedef {{ selector: string, hasParent: boolean, quasiCode: Array<{ insideCssValue: boolean, code: string }>, cssPartExpressions: { [key: number]: CssPartExpression[]} }} CssPartExpression
@@ -53,7 +54,7 @@ module.exports = async function cssLoader(source) {
     configFile: false,
     plugins: [
       [
-        "@babel/plugin-syntax-typescript",
+        babelPluginTypeScript,
         { isTSX: this.resourcePath.endsWith(".tsx") },
       ],
     ],
diff --git a/node_modules/next-yak/loaders/tsloader.cjs b/node_modules/next-yak/loaders/tsloader.cjs
index 8196470..6c05517 100644
--- a/node_modules/next-yak/loaders/tsloader.cjs
+++ b/node_modules/next-yak/loaders/tsloader.cjs
@@ -2,6 +2,8 @@
 const babel = require("@babel/core");
 const getYakImports = require("./lib/getYakImports.cjs");
 const InvalidPositionError = require("./babel-yak-plugin.cjs").InvalidPositionError;
+const plugin = require("./babel-yak-plugin.cjs")
+const babelPluginTypeScript = require('@babel/plugin-syntax-typescript');
 
 /**
  * Loader for typescript files that use yak, it replaces the css template literal with a call to the 'styled' function
@@ -48,12 +50,12 @@ module.exports = async function tsloader(source) {
       filename: resourcePath,
       configFile: false,
       plugins: [
+          [
+            babelPluginTypeScript,
+            { isTSX: this.resourcePath.endsWith(".tsx") },
+          ],
         [
-          "@babel/plugin-syntax-typescript",
-          { isTSX: this.resourcePath.endsWith(".tsx") },
-        ],
-        [
-          require.resolve("./babel-yak-plugin.cjs"),
+          plugin,
           {
             replaces,
             rootContext,
diff --git a/node_modules/next-yak/package.json b/node_modules/next-yak/package.json
index c4c788f..8da6bda 100644
--- a/node_modules/next-yak/package.json
+++ b/node_modules/next-yak/package.json
@@ -28,7 +28,9 @@
         "import": "./dist/context/index.js",
         "require": "./dist/context/index.cjs"
       }
-    }
+    },
+    "./loaders/tsloader": "./loaders/tsloader.cjs",
+    "./loaders/cssloader": "./loaders/cssloader.cjs"
   },
   "scripts": {
     "prepublishOnly": "npm run build && npm run test && npm run test:types:code && npm run test:types:test",
